plugins/com.google.cloud.tools.eclipse.projectselector/META-INF/MANIFEST.MF

```diff
@@ -40,6 +40,7 @@ Import-Package: com.google.api.client.auth.oauth2,
  org.eclipse.jface.dialogs,
  org.eclipse.jface.layout,
  org.eclipse.jface.viewers,
+ org.eclipse.jface.window,
```

Comment:
It's needed by the Dialog class. Without it we don't get error in Eclipse, but we get a warning that `Window.open()` is restricted and we shouldn't use it. This makes the dependency explicit and Eclipse happy.